### PR TITLE
Fix several README links for `url-check` tool

### PR DESCRIPTION
**Pull request by elliot42**
A variety of links related to the `check-ia` / `url-check` tool (the terms are used inconsistently) were broken in the READMEs.  This PR fixes several links and removes some trailing whitespace.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,32 @@ A collection of code snippets mostly designed to be dropped into the data harves
...
   cp -r harvesting-tools/TOOLNAME/* RESOURCEUUID/tools/
   ```
 - Adjust the base URL for the resource along with any other relevant variables, and tweak the content of the tool as necessary
-- After the resource has been harvested, proceed with the further steps in the [workflow](https://github.com/datarefugephilly/workflow/). 
+- After the resource has been harvested, proceed with the further steps in the [workflow](https://github.com/datarefugephilly/workflow/).
 
 ## Matching Tools and Datasets
 
-Each tool in this repo has a fairly specific use case. Your choice will depend on the shape and size of the data you're dealing with. Some datasetswill require more creativity/more elaborate tools. If you write a new tool, please [add it to the repo](#contributing). 
+Each tool in this repo has a fairly specific use case. Your choice will depend on the shape and size of the data you're dealing with. Some datasetswill require more creativity/more elaborate tools. If you write a new tool, please [add it to the repo](#contributing).
```
Comment (dcwalk) [inline README.md]: I think this link might need to be changed to `./CONTRIBUTING.md`? probably a relic of the previous iteration

**Review by dcwalk — COMMENTED** [review]
Looks great, thanks for the catch, if you are able to make that quick change to contributing too that would be great :)

File: README.md (hunk 1 above)
Comment (elliot42) [reply README.md]: Link fixed thanks

**Comment by elliot42** [comment]
Updated link to CONTRIBUTING.md

**Review by dcwalk — APPROVED** [review]

**Pull request merged**